packages/@aws-cdk/aws-servicecatalogappregistry/lib/application.ts

```diff
@@ -102,7 +149,9 @@ export class Application extends ApplicationBase {
    * Imports an Application construct that represents an external application.
    *
    * @param scope The parent creating construct (usually `this`).
+   *
    * @param id The construct's name.
+   *
    * @param applicationArn the Amazon Resource Name of the existing AppRegistry Application
```

Comment:
I should have been more clear in my last comment about these, my bad:
```suggestion
   * @param scope The parent creating construct (usually `this`).
   * @param id The construct's name.
   * @param applicationArn the Amazon Resource Name of the existing AppRegistry Application
```